### PR TITLE
battery disabler -> master

### PR DESCRIPTION
**Pull request by NoamRosenberg08**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/commands/swerve/Battery/BatteryLimiter.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    @Override
+    public void execute() {
+        double currentAverageVoltage;
+
+        if(battery.getTotalCurrent() <= USAGE_MIN_LEVEL) {
+            currentAverageVoltage = voltageFilter.calculate(battery.getCurrentVoltage());
+
+
+            if (currentAverageVoltage <= battery.getMinimumVoltage()
+                    && DriverStation.getMatchType() == DriverStation.MatchType.None) {
```
Comment (Stormageddon37) [inline src/main/java/edu/greenblitz/robotName/commands/swerve/Battery/BatteryLimiter.java:41]: Does this `MatchType.None` mean practice?

File: src/main/java/edu/greenblitz/robotName/commands/swerve/Battery/BatteryLimiter.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @Override
+    public void initialize() {
+        for (int i = 0; i < LEN_OF_AVG; i++) {
+            voltageFilter.calculate(battery.getCurrentVoltage());
+        }
+    }
+
+    @Override
+    public void execute() {
+        double currentAverageVoltage;
```
Comment (Stormageddon37) [inline src/main/java/edu/greenblitz/robotName/commands/swerve/Battery/BatteryLimiter.java:34]: Why is this needed if the value is only initialized AND used inside the `if` block?

**Review by Stormageddon37 — CHANGES_REQUESTED** [review]
Small PR 🤯

File: src/main/java/edu/greenblitz/robotName/commands/swerve/Battery/BatteryLimiter.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+public class BatteryLimiter extends GBCommand {
+
+    private static final int WINDOW_SIZE = 50;
+    private Battery battery;
+    private LinearFilter voltageFilter;
+
+
+    public BatteryLimiter() {
+        require(battery);
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/commands/swerve/Battery/BatteryLimiter.java:16]: Are you sure this will work?

**Review by greenblitz4590 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

**Pull request merged**